library/Ivoz/Provider/Domain/Service/Company/ValidateBillingMethod.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Ivoz\Provider\Domain\Service\Company;
+
+use Ivoz\Core\Application\Service\EntityTools;
+use Ivoz\Provider\Domain\Model\Company\CompanyDto;
+use Ivoz\Provider\Domain\Model\Company\CompanyInterface;
+use Ivoz\Provider\Domain\Model\Feature\Feature;
+
+class ValidateBillingMethod implements CompanyLifecycleEventHandlerInterface
```

Comment:
I think that SanitizeBillingMethod would be a better class name. I was expecting you to throw an exception on company "validation"

Reply:
Renamed! :+1: